src/styles/index.ts

```diff
@@ -5929,6 +5929,26 @@ const styles = (theme: ThemeColors) =>
             aspectRatio: 1.7,
         },
 
+        receiptPreview: {
+            position: 'absolute',
+            left: 60,
+            top: 100,
+            width: 400,
+            height: 600,
+            borderRadius: 24,
+            overflow: 'hidden',
+        },
+
+        receiptPreviewImageWrapper: {
```

Comment:
both this and `receiptPreviewImage` can be removed because they are so simple.
Just apply w100 and h100 classes directly in the component